src/components/Pill/index.jsx

```diff
@@ -39,9 +39,6 @@ const Pill = ({ statuses }) => {
           text: 'In review',
           className: 'badge badge-light',
         });
-        pills.push({
-          text: <i className="fa fa-lock" />,
-        });
         break;
```

Comment:
although this was in design, the lock has some semantical visual implications. In my opinion, let's keep the lock for now. But if you have a feedback on it, I am open to it.

Reply:
i personally like the lock too...i have brought it back